templates/templates/BenchmarkDotNet.BenchmarkProjectTemplate.CSharp/.template.config/template.json

```diff
@@ -98,7 +144,7 @@
       "type": "parameter",
       "datatype": "string",
       "description": "Version of BenchmarkDotNet that will be referenced.",
-      "defaultValue": "0.12.1",
```

Comment:
With each new release, you need to change this manually.

Currently, if you add BDN.Templates, a project will be created with BDN v0.12.1.

Reply:
I try to handle this issue – https://github.com/dotnet/BenchmarkDotNet/pull/1879
 @AndreyAkinshin could you take a look please ?